app/__init__.py

```diff
@@ -29,6 +29,8 @@ def handleBaseException(error):
 from app.todo_lists.models import TodoList
 from app.todos.models import Todo
 from app.users.resources import UserListResource, UserResource
+from app.todos.resources import todoMultiResource, todoResource
```

Comment:
Class names should use `CapWords` convention instead of `camelCase`. s/todoMultiResource/TodoMultiResource and s/todoResource/TodoResource